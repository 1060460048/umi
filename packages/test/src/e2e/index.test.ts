test('with img', async () => {
  await page.goto('https://umijs.org/docs/how-umi-works');

  await page.setFamily();
  await waitTime(1000);
  const image = await page.fullPageScreenshot();

  expect(image).toMatchImageSnapshot({
    dumpDiffToConsole: true,
    failureThreshold: 0.2,
    blur: 10,
  });
});
