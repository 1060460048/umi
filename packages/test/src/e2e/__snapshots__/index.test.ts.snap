// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page test 1`] = `"UmiJSDocsConfigAPIGuidePluginsv2.xGitHub‰∏≠ÊñáUmiJSüçôExtensibleenterprise-levelfront-endapplicationframework.GetStarted‚ÜíExtensibleUmiimplementsthecompletelifecycleandmakesitextensible,andUmi'sinternalfunctionsareallplugins.Umialsosupportpluginsandpresets.OutoftheBoxUmihasbuilt-inrouting,building,deployment,testing,andsoon.Itonlyrequiresonedependencytogetstarted.UmialsoprovidesanintegratedpresetforReactwithrichfunctionaries.EnterpriseIthasbeenverifiedby3000+projectsinAntFinancialandprojectsofAlibaba,Youku,Netease,È£ûÁå™,KouBeiandothercompanies.SelfDevelopmentIncludingmicrofrontendlibrary,componentpackaging,documentationtools,requestlibrary,hookslibrary,dataflow,etc.PerfectRoutingSupportsbothconfigurationroutingandconventionrouting,whilewithfunctionalcompleteness,suchasdynamicrouting,nestedrouting,permissionrouting,andsoon.FutureReadyUmi'scommunityisalsoexploringnewtechnologies.Forexample,modernmode,webpack@5,automatedexternal,bundlerless,etc.GettingStartedin3minutesManually,#Createdirectory$mkdirmyapp&&cdmyapp#Installdependency$yarnaddumi#Createpage$npxumigpageindex--typescript--less#Startdevelopment$npxumidevOrGettingStartedwithBoilerplate.ContributorsThisprojectexiststhankstoallthepeoplewhocontribute.Joinus!FeedbackGithubIssueDingTalkWechatumijs/umi/issuesOpen-sourceMITLicensed|Copyright¬©2017-presentPoweredbydumi."`;
